staging/src/k8s.io/kubectl/pkg/cmd/completion/completion.go

```diff
@@ -89,13 +89,17 @@ var (
 		# Load the kubectl completion code for zsh[1] into the current shell
 		    source <(kubectl completion zsh)
 		# Set the kubectl completion code for zsh[1] to autoload on startup
-		    kubectl completion zsh > "${fpath[1]}/_kubectl"`))
+		    kubectl completion zsh > "${fpath[1]}/_kubectl"
+
+		# Load the kubectl completion code for powershell into the current shell
+			kubectl completion powershell | Out-String | Invoke-Expression`))
```

Comment:
nit: Could you make the indentation depth consistent with the above text?

Reply:
just out of curiosity, could we save the completion code to a dedicated file so that powershell could load the code on startup?

Reply:
This just came from cobra's example here: https://github.com/spf13/cobra/blob/master/shell_completions.md#creating-your-own-completion-command.
You can save the completion string to a .ps1 file, but you still need to run it from the $PROFILE. I've typically just shoved the command into my $PROFILE to make the mobilty a bit easier.

Reply:
Corrected the indentation.
I also added some more options for auto-loading the powershell completion code on startup.

Reply:
Great!